pyccl/halos/halo_model.py

```diff
@@ -988,6 +988,163 @@ def halomod_Tk3D_1h(cosmo, hmc,
     return tk3d
 
 
+def halomod_Tk3D_SSC_linear_bias(cosmo, hmc, prof, bias1=1, bias2=1, bias3=1,
+                                 bias4=1,
+                                 is_number_counts1=False,
+                                 is_number_counts2=False,
+                                 is_number_counts3=False,
+                                 is_number_counts4=False,
+                                 p_of_k_a=None, lk_arr=None,
+                                 a_arr=None, extrap_order_lok=1,
+                                 extrap_order_hik=1, use_log=False):
+    """ Returns a :class:`~pyccl.tk3d.Tk3D` object containing
+    the super-sample covariance trispectrum, given by the tensor
+    product of the power spectrum responses associated with the
+    two pairs of quantities being correlated. Each response is
+    calculated as:
+
+    .. math::
+        \\frac{\\partial P_{u,v}(k)}{\\partial\\delta_L} =
+        \\left(\\frac{68}{21}-\\frac{d\\log k^3P_L(k)}{d\\log k}\\right)
+        b_u b_v P_L(k)+I^1_2(k|u,v) - (b_{u} + b_{v})
+        P_{u,v}(k)
```

Comment:
I think this is not correct, right? b_u and b_v multiply everything

Reply:
Fixed.